Sources/TuistSupport/Utils/FileHandler.swift

```diff
@@ -293,20 +293,7 @@ public class FileHandler: FileHandling {
 
     public func urlSafeBase64MD5(path: AbsolutePath) throws -> String {
```

Comment:
This looks good to me, yet I wonder why we are not using our hashing methods instead? 🤔